src/test/scala/chiselTests/StrongEnum.scala

```diff
@@ -135,7 +135,7 @@ class StrongEnumFSM extends Module {
     is (sOne1) {
       when (io.in) {
         state := sTwo1s
-      } .otherwise {
+      }.otherwise {
```

Comment:
typo?

Reply:
The old one was a typo. I personally think it looks better without the space.

Reply:
Fine with me though I believe the typical style has been to treat `otherwise` like `else` since Chisel is a DSL, so preferring

```scala
when (a) {
  // ...
} .otherwise {
  // ...
}
```

in analogy with

```scala
if (foo) {
  // ...
} else {
  // ...
}
```

Maybe the others have some thoughts on this as well.

Reply:
Oh, I wasn't aware of that. I've actually been using the `}.otherwise {` style everywhere else. I can just change them though, so that the style is consistent

Reply:
It turns out there was only one `.otherwise`, which I've changed to have a space.